addons/bitbucket/models.py

```diff
@@ -193,6 +193,11 @@ def is_private(self):
         connection = BitbucketClient(access_token=self.api.fetch_access_token())
         return connection.repo(user=self.user, repo=self.repo)['is_private']
 
+    @property
+    def fetch_repo(self):
```

Comment:
I wasn't clear, it's the verb tense that makes this confusing. So prefer something like fetched_repo.